site/content/docs/v1.4/self-signed-certificates.md

```diff
@@ -32,3 +32,17 @@ the `--cacert` flag to provide a path to the certificate to be trusted.
 ```bash
 velero backup describe my-backup --cacert <PATH_TO_CA_BUNDLE>
 ```
+
+## Error with client certificate with custom S3 server
+
+In case you are using a custom S3-compatible server, you may encounter that the backup fails with error similar to one below -
```

Comment:
nit: update to "you may encounter that the backup fails with an error similar to one below."

for all the files

Reply:
Done in next patchset.